### DISCUSSION TITLE
CLI: get schema by ID

### DISCUSSION THREAD

**Discussion by wkalt** [foxglove/mcap#1586] [General]
Currently a user can view schemas in an mcap file with `mcap list schemas`. However, it is difficult to actually extract a schema from the output since it is rendered in table form.

It would be useful to also support an `mcap get schema <id>` that returns just the schema for that ID. Questions to consider:
* Is `get` the verb we want to go with? This should match what we do for attachments as well I expect.
* Should this command also return the rest of the metadata for that schema in some way? Or perhaps it should render the table view by default, and support a --schema-only flag?

**Comment by jhurliman** [comment]
Is this related to #402 ?

**Comment by jhurliman** [comment]
`mcap get attachment` and `mcap get metadata` both exist now. Next, we need `mcap get schema` and possibly `mcap get channel`.
